packages/code-infra/src/cli/cmdBuild.mjs

```diff
@@ -442,5 +463,14 @@ export default /** @type {import('yargs').CommandModule<{}, Args>} */ ({
       outputDir: buildDir,
       addTypes: buildTypes,
     });
+
+    await copyHandler({
+      cwd,
+      files: args.copy ?? [],
+      globs: args.copyGlob ?? [],
+      buildDir: buildDirBase,
+      excludeDefaults: args.skipCopyDefaultFiles || false,
+      silent: !!args.verbose,
```

Comment:
This doesn't seem right

```suggestion
      silent: !args.verbose,
```

but better yet, just use the same nomenclature everywhere
```suggestion
      verbose: args.verbose,
```
or
```suggestion
      silent: args.silent,
```